ui/src/common/QualityInfo.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Chip from '@material-ui/core/Chip'
+
+const useStyles = makeStyles((theme) => ({}))
+
+export const QualityInfo = (props) => {
+  const classes = useStyles()
+  let { suffix, bitRate } = props.song ? props.song : props.record
```

Comment:
The property should always be `record`, not `song`. Then you could use deconstruction to get the `suffix` and `bitRate`, and pass the `rest` to the `Chip` component. Search the codebase for some examples.